cpp/ql/src/semmlecode.cpp.dbscheme

```diff
@@ -705,6 +705,11 @@ usertype_uuid(
     unique string uuid: string ref
 );
 
+mangled_type_name(
+    unique int id: @usertype ref,
```

Comment:
Why limit mangled names to usertypes? Functions and variables also get mangled names, so it could be any `@declaration`.